tests/lib/numbers.js

```diff
@@ -0,0 +1,51 @@
+var addon = require('../native');
+var assert = require('chai').assert;
+
+describe('JsNumber', function() {
+  it('return a JsNumber built in Rust', function () {
+    assert.equal(9000, addon.return_js_number());
```

Comment:
I think the API is `equal(actual, expected)`:

http://chaijs.com/api/assert/#method_equal